ksqldb-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedStream.java

```diff
@@ -67,7 +67,8 @@
       final List<FunctionCall> aggregations,
       final Optional<WindowExpression> windowExpression,
       final FormatInfo valueFormat,
-      final QueryContext.Stacker contextStacker
+      final Stacker contextStacker,
+      final KsqlConfig config
```

Comment:
This `config` is not used here, right?  I see `SchemaKTable` is created with `ksqlConfig`.

Reply:
good catch, I used it in `SchemaKGroupedTable` because I didn't realize it had access to the field `ksqlConfig`. I removed it